mbed/src/iot_network.cpp

```diff
@@ -88,11 +88,12 @@ struct NetworkConnection {
             this->mtx.unlock();
             // XXX: I don't think this really safe. cbk & ctx might be freed between the unlock
             // and this call
+            // Note: Currently cbk is always null because the IotMqtt library does not call setCloseCallback
             if (cbk != NULL) {
                 cbk(this, IOT_NETWORK_UNKNOWN_CLOSED, ctx); 
             }
         }
-        tr_debug("exiting dispatcher thread.");
+        tr_info("Exiting event_dispatcher_thread");
```

Comment:
```suggestion
        tr_debug("Exiting event_dispatcher_thread");
```
Traces can default to debug verbosity?

Reply:
Given the unstability around disconnection, I thought that this would be a nice information message. As it is only printed once, it does not overflow the console.